client/windows/settings/GeneralOptionsTab.cpp

```diff
@@ -190,6 +194,9 @@ GeneralOptionsTab::GeneralOptionsTab()
 	std::shared_ptr<CToggleButton> framerateCheckbox = widget<CToggleButton>("framerateCheckbox");
 	framerateCheckbox->setSelected(settings["video"]["showfps"].Bool());
 
+	std::shared_ptr<CToggleButton> hapticFeedbackCheckbox = widget<CToggleButton>("hapticFeedbackCheckbox");
+	hapticFeedbackCheckbox->setSelected(settings["general"]["hapticFeedback"].Bool());
```

Comment:
This checkbox might not exist, e.g. on desktop systems
```suggestion
	if (hapticFeedbackCheckbox)
		hapticFeedbackCheckbox->setSelected(settings["general"]["hapticFeedback"].Bool());
```